### PR TITLE
Change button checkbox/radios to ignore hidden input fields

### PR DESCRIPTION
**Pull request by mvastola**
Allows for HTML like:
```html
<div class="btn-group-toggle" data-toggle="buttons">
  <label class="btn btn-info">
    <input name="checkbox" type="hidden" value="0">
    <input id="cb" name="checkbox" type="checkbox" value="1" autocomplete="off"> Click Me
  </label>
</div>
```

Fixes twbs/bootstrap#27795

### CODE REVIEW

**Comment by mvastola** [comment]
NB: I wasn't sure if I was supposed to do `npm run dist` for my PR. I opted not to because, upon running it, I saw changes were being applied that weren't due to this patch. Please let me know if I should have run this.

**Comment by XhmikosR** [comment]
Why not just exclude `input`s with `type=hidden`?

**Comment by mvastola** [comment]
@XhmikosR, oh. I'm sorry. I was on mobile and for some reason it didn't show the "Member" tag. I'll go and change it.

File: js/src/button.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ const Selector = {
   DATA_TOGGLE_CARROT : '[data-toggle^="button"]',
   DATA_TOGGLE        : '[data-toggle="buttons"]',
   INPUT              : 'input',
```
Comment (XhmikosR) [inline js/src/button.js]: FYI you made this unused with the current patch.

**Comment by mvastola** [comment]
Oh. I left it because I wasn't sure if it was used elsewhere and forgot to check. I'll remove. Just pushed the change in the attribute selector.

**Comment by mvastola** [comment]
Done

**Review by Johann-S — APPROVED** [review]

**Pull request merged**